shoov/modules/custom/shoov_ui_build/drush/shoov_ui_build.drush.inc

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * @file
+ * Drush commands for UI Builds.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function shoov_ui_build_drush_command() {
+  $items = array();
+
+  $items['delete-old-ui-builds'] = array(
+    'description' => 'Delete old UI builds and screenshots.',
+  );
+
+  return $items;
+}
+
+/**
+ * Command callback for "delete-old-ui-builds".
+ *
+ * Deletes ui-builds that are older than one month, meanwhile keeping 10 builds
+ * in each repository.
+ */
+function drush_shoov_ui_build_delete_old_ui_builds() {
+
+  $query = new EntityFieldQuery();
+  $result = $query
+    ->entityCondition('entity_type', 'node')
+    ->entityCondition('bundle', 'repository')
+    ->propertyCondition('status', NODE_PUBLISHED)
+    ->propertyOrderBy('nid', 'ASC')
+    ->execute();
+
+  if (empty($result['node'])) {
+    // Stop if there's not repositories.
+    drush_log('No reposirories found', 'warning');
+    return;
+  }
+
+  $ids = array_keys($result['node']);
+  $repos = node_load_multiple($ids);
+  foreach ($repos as $repo) {
+    drush_print('Checking Repository "' . $repo->title . '"');
+
+    // Need to use db_select() because field condition override in OG doesn't
+    // work in a Drush command.
+    $ui_build_query = db_select('og_membership', 'm');
```

Comment:
I need all the builds inside a specific repository,
I couldn't use `entityFieldQuery` because the field condition override in OG (Which allows to take the relation from `og_membership`) didn't work when running this function, Ran fine in `devel_init` for example.
This is why I had to use `db_select`